README.md

```diff
@@ -12,7 +12,8 @@ To better support Instructors of Large Courses in Canvas,  given a CourseId we s
 This utility is written in python and takes `:course_id`, `:group_category_name` from the property file and creates a group category with the name provided in the property file. We will get all the sections and users
 in the course and create one group for each section and add all the users in each section to the corresponding group. The group name will be identical to the section name
  
-## Requirements 
+
+### Running the script with out openshift
 
 1. Python version > = 2.7.10 is required.
```

Comment:
I think it should run, but i never tried it. In the `Dockerfile` i have put the ubuntu16.04 defaults to Python 2.7.12